charts/csm-installer/values.yaml

```diff
@@ -33,10 +33,10 @@ dbPort: 26257
 dbSSLEnabled: "true"
 
 # CSM Installer image
-installerImage: dellemc/dell-csm-installer:v1.0.2
+installerImage: dellemc/dell-csm-installer:v1.0.1
```

Comment:
Are we downgrading this?

Reply:
There is no v1.0.2 tagged image. This chart was incremented in the 1.10.2 release but this chart is no longer actively maintained so this tag should not have been updated.